oceanmodes/test/test_modes.py

```diff
@@ -264,3 +266,114 @@ def test_Eady(self, atol=5e-2, nz=20, Ah=0.):
+           1.08620176e-04,8.89353316e-05,7.59696940e-05,6.85660114e-05,
+           6.13053056e-05,4.96623307e-05,4.26153730e-05,3.83933836e-05,
+           3.33244718e-05,3.14573673e-05,2.74780120e-05,2.50869570e-05,
+           2.35907064e-05,2.26047540e-05,2.06309696e-05,1.93499569e-05,
+           1.85802260e-05,1.73887368e-05,1.49995125e-05,1.25945779e-05,
+           1.06210281e-05,9.69791903e-06,9.18845407e-06,7.56655382e-06,
+           5.00383224e-06,2.83633215e-06,1.67170878e-06,1.19350982e-06,
+           9.87252978e-07,9.54096555e-07,1.05979705e-06,1.18957478e-06,
+           1.23432375e-06,1.22190029e-06,1.20084648e-06,1.18287498e-06,
+           1.14801849e-06,1.08643614e-06,9.36511867e-07,6.17342678e-07,
+           3.35507108e-07,3.19646379e-07,2.60733349e-07,np.nan,
+            np.nan])
+        f0 = 9.27671062527e-05
+        beta = 1.76637107718e-11
+        Nx = 100
+        Ny = 100
+        dx = 1e-1
+        dy = 1e-1
+        k = fft.fftshift( fft.fftfreq(Nx, dx) )
+        l = fft.fftshift( fft.fftfreq(Ny, dy) )
+        k = k[np.absolute(k) < 5.*Rd**-1]
+        l = l[np.absolute(l) < 5.*Rd**-1]
+        etax = np.zeros(2)
+        etay = np.zeros(2)
+
+        z, growth_rate, vertical_modes = \
+            modes.instability_analysis_from_N2_profile(
+                z_N2, N2, f0, beta, k, l,
+                z_u, ubar, vbar, etax, etay, sort='LI', num=2
+        )
+        
+        nz = len(z)
+
+        ###################
+        # make sure we got the right number of modes
+        # NOPE! don't want all the modes and scipy.sparse.linalg.eigs won't
+        # compute them
+        # self.assertEqual(nz, len(def_radius))
+        # make sure the modes themselves have the right structure
+        ###################
+        self.assertEqual(nz, vertical_modes.shape[0],
+            msg='modes array must be in the right shape')
+
+        self.assertTrue(np.all( np.diff(
+                    growth_rate.reshape((growth_rate.shape[0], len(k)*len(l))).imag.max(axis=1) ) <= 0.),
+        #self.assertTrue(np.all( max_growth_rate == 0.),
+            msg='imaginary part of modes should be descending')
+
+        #nmodes = len(def_radius)
+        #zero_crossings = np.abs(np.diff(np.sign(bc_modes), axis=0)/2).sum(axis=0)
+        #self.assertListEqual(list(range(nmodes)), list(zero_crossings),
+        #    msg='modes should have the correct number of zero crossings')
+
+        mode_amplitude1 = (np.absolute(vertical_modes[:, 0])**2).sum(axis=0)
+        self.assertTrue(np.allclose(1., mode_amplitude1),
+            msg='mode1 should be normalized to amplitude of 1 at all horizontal wavenumber points')
+
+        mode_amplitude2 = (np.absolute(vertical_modes[:, 1])**2).sum(axis=0)
+        self.assertTrue(np.allclose(1., mode_amplitude2),
+            msg='mode2 should be normalized to amplitude of 1 at all horizontal wavenumber points')
```

Comment:
The OCCA data files were also added by mistake. They have already been deleting from git by the command 'git rm --cached' so I currently don't know how to delete the binary files. Maybe we could delete them from the Github web-based edit after we merge.

I do not know why travis build did not work.

As for the actual values of the growth rates, the profiles used do not have an analytical solution so I don't have anything to compare them with.